run.py

```diff
@@ -200,7 +200,11 @@ def add_optimizer(self, parameters):
 		Returns an optimizer for learning the parameters of the model
 		
 		"""
-		return torch.optim.Adam(parameters, lr=self.p.lr, weight_decay=self.p.l2)
+		if self.p.optimizer == 'Adagrad' :
+			print(f'Using optimizer {self.p.optimizer}')
```

Comment:
Maybe print this in general and not only when using Adagrad opimizer.

Reply:
This was to test if the selections works. We dont need to print it, because it is included in the parameters.